features/user/account/account_password.feature

```diff
@@ -20,7 +20,7 @@ Feature: User account password change
           And I fill in "Confirmation" with "newpassword"
           And I press "Save changes"
          Then I should still be on my account password page
-          And I should see "Wrong current password"
+          And I should see "Wrong current password."
```

Comment:
IIRC we agreed that matching texts should not contain dots etc.

@aRn0D @pjedrzejewski @umpirsky ping